isisd/isis_spf.c

```diff
@@ -1403,25 +1403,23 @@ int isis_spf_schedule(struct isis_area *area, int level)
 			"ISIS-Spf (%s) L%d SPF schedule called, lastrun %d sec ago",
 			area->area_tag, level, diff);
 
+	if (area->spf_timer[level - 1])
+		return ISIS_OK;
+
```

Comment:
I believe this change is incorrect. When we are using draft-ietf-rtgwg-backoff-algo, `spf_backoff_schedule` needs to be called again, to update the timers/state for the backoff algo, even when a timer is already pending.

I tried to point that out in the comment directly following the return statement newly introduced, apperently the comment was not clear enough.